distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -66,30 +48,128 @@ a|* Does not scale. Only runs one single server instance.
 a|* Can be embedded in Java easily.
 * Requires no HTTP connection.
 * Uses the same interface as the Standalone Solr Server uses under the covers.
-* Allows for full control over the Solr Server. No synchronous issues on startup; i.e., the Solr Server will synchronously start up with the Solr Catalog Provider
-* Runs within the same JVM
+* Allows for full control over the Solr Server. No synchronous issues on startup; i.e., the Solr Server will synchronously start up with the Solr Catalog Provider.
+* Runs within the same JVM.
 * Setup and Installation is simple: unzip and run.
-|* Can only be interfaced using Java
+a|* Can only be interfaced using Java.
 
 |(Administrative)Tools
-a|* External open source tools like Luke will work to allow admins to check index contents
-* JMX metrics reporting is enabled
+a|* External open source tools like Luke will work to allow admins to check index contents.
+* JMX metrics reporting is enabled.
 
 a|* No Web Console. 
 * No easy way to natively access (out of the box) what is in the index files or health of server at the data store level.
 
 |Security
 a|* Does not open any ports which means no ports have to be secured.
-|
+a|
+ 
+|Performance
+a|* Requires no HTTP or network overhead.
+* Near real-time indexing.
+* Can understand complex queries.
+a|
+
+|Backup/Recovery
+a|* Can manually or through custom scripts back up the indexes.
+a|* Must copy files when server is shutdown.
+
+|===
+
+==== Solr Catalog External Provider
+
+Use the Solr Catalog External Provider if using the ${branding} in production mode.
+The Solr Catalog External Provider is a standalone Solr Server used on a separate machine.
+The Solr Catalog External Provider provides a Solr Admin GUI or a "REST-like HTTP/XML and JSON API."
+
+.Solr Catalog External Provider Pros and Cons
+[cols="1,4a,2a" options="header"]
+|===
+
+|Feature
+|Pro
+|Con
+
+|Scalability
+a|* Allows the middle-tier to be scaled by pointing various middle-tier instances to one server facade.
+* Possible data tier scalability with Solr Cloud. Solr Cloud allows for "high scale, fault tolerant, distributed indexing and search capabilities.
+a|
+
+|Flexibility
+a|* REST-like HTTP/XML and JSON APIs that make it easy to use from virtually any programming language.
+* Ability to run in separate or same JVM of middle tier.
+a|
```

Comment:
I can add that too, also @lessarderic should I move the Solr Catalog External Provider and Solr Cloud to solr-http-catalog-provider-contents.adoc instead?